demo/src/index.js

```diff
@@ -1,7 +1,6 @@
 import React, { useState, useEffect, useCallback } from "react";
 import ReactDOM from "react-dom";
-import ColorPicker from "../../src/hex";
-import HexInput from "../../src/HexInput";
+import { HexColorPicker, HexInput } from "../../src";
```

Comment:
**Discussion: Shouldn't we change the component names.**

These names seem inconsistent to me.
```
HexColorPicker
HexInput
```

At the same time, names like `RgbStringColorPicker` are too long IMO.

What do you think @RyanChristian4427. Is that important to have `Color` here or not?

Renaming everything this way feels right to me:
```
HexPicker
HexInput
RgbPicker
RgbStringPicker
...
```

react-color exports things like `CompactPicker`, `GithubPicker, `HuePicker`, etc. So I guess it's okay in terms of DX.

Reply:
Hm, I'd say I'm in favour of keeping `Color` in the names personally. While yes it adds some length, it also reduces ambiguity to 0. 

If I come across code that references a `HexColorPicker` I'd know and understand what that is instantly. `HexPicker` maybe not, at least until I look at the import to see where it's coming from.

Reply:
So I'd also be in favour of `HexColorInput`, but that's just me.

Reply:
I think you're right.
Let's rename `HexInput` to `HexColorInput` then?

Reply:
Done